### PR TITLE
SAAS 835 - Sign Up Tests

### PR DESCRIPTION
**Pull request by peterSirotnak**
[SAAS-835](https://jira.percona.com/browse/SAAS-835) - Sign Up Tests.

### CODE REVIEW

File: cypress/integration/auth/signUp.spec.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        newUser.email = signUpPage.methods.getMailosaurEmailAddress(newUser);
+        signUpPage.methods.fillOutSignUpForm(newUser);
+        cy.get(signUpPage.locators.termsOfService).check({force: true});
+        cy.get(signUpPage.locators.registerButton).isEnabled().click();
+        cy.mailosaurGetMessage(
+            mailosaurServerId,
+            {sentTo: newUser.email},
+            {timeout: 20000},
+        ).then((message) => {
+            expect(message.html.body).to.contain(`Hi ${newUser.firstName},`);
```
Comment (yurkovychv) [inline cypress/integration/auth/signUp.spec.js:162]: Could you please add more verifications for the email ? Like title and some other wordings?

File: cypress/integration/auth/signUp.spec.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        signUpPage.methods.verifyValidationSignUp({email: true});
+        cy.reload();
+        signUpPage.methods.fillOutSignUpForm(newUser);
+        cy.get(signUpPage.locators.termsOfService).check({force: true});
+        cy.get(signUpPage.locators.registerButton).isEnabled();
+        cy.get(signUpPage.locators.termsOfService).uncheck({force: true});
+        cy.get(signUpPage.locators.registerButton).isDisabled();
+    });
+
+    it('SAAS-T122 - Verify confirmation email is sent during Portal registration', () => {
```
Comment (yurkovychv) [inline cypress/integration/auth/signUp.spec.js:152]: I think this is covered by SAAS-T78

File: cypress/integration/auth/signUp.spec.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+            mailosaurServerId,
+            {sentTo: newUser.email},
+            {timeout: 20000},
+        ).then((message) => {
+            expect(message.html.body).to.contain(`Hi ${newUser.firstName},`);
+            cy.mailosaurDeleteMessage(message.id);
+        });
+    });
+
+    it('SAAS-T124 - Verify user can see notification about Account confirmation email sent', () => {
```
Comment (yurkovychv) [inline cypress/integration/auth/signUp.spec.js:167]: This is covered too by SAAS-T78

File: cypress/pages/auth/signUp.page.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+      inputPasswordChildren: '[type="password"]',
+      inputFirstName: '[data-se="o-form-input-firstName"]',
+      inputLastName: '[data-se="o-form-input-lastName"]',
+      registerButton: '[type="submit"]',
+      termsOfService: '[name="tos"]',
+      termsOfServiceLabel: '[data-se-for-name="tos"]',
+      signInLink: '[name="back-link]',
+      registrationAlert: '[role="alert"]',
+      registrationCompleteContainer: '[data-se="o-form-content"]',
+      registrationCompleteTitle: '[class="title"]',
```
Comment (yurkovychv) [inline cypress/pages/auth/signUp.page.ts:47]: `.title`

File: cypress/pages/auth/signUp.page.ts [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+      registrationCompleteDesc: '[class="desc"]',
```
Comment (yurkovychv) [inline cypress/pages/auth/signUp.page.ts:48]: `.desc`

File: cypress/integration/auth/signUp.spec.js (hunk 3 above)
Comment (peterSirotnak) [reply cypress/integration/auth/signUp.spec.js:167]: Okay, i have added log to the SAAS-T78, and removed SAAS-124

**Review by yurkovychv — APPROVED** [review]

**Comment by yurkovychv** [comment]
Great work @peterSirotnak

**Comment by peterSirotnak** [comment]
> Great work @peterSirotnak

Thank you @yurkovychv

**Pull request merged**